e2e-tests/playwright/tests/functional/channels/message_priority/standard_priority.spec.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect, test} from '@e2e-support/test_fixture';
+
+test('MM-T5139: Message Priority - Standard message priority and system setting', async ({pw, pages}) => {
+    // # Setup test environment
+    const {user} = await pw.initSetup();
+    
+    // # Log in as a user in new browser context
+    const {page} = await pw.testBrowser.login(user);
+
+    // # Visit default channel page
+    const channelPage = new pages.ChannelsPage(page);
+    await channelPage.goto();
+    await channelPage.toBeVisible();
+
+    // # Verify formatting bar has message priority icon
+    await expect(channelPage.centerView.postCreate.priorityButton).toBeVisible();
+
+    // # Click on the message priority icon and verify menu
+    await channelPage.centerView.postCreate.openPriorityMenu();
+    
+    // # Verify menu opens with correct header
+    const priorityDialog = page.getByRole('dialog');
+    await expect(priorityDialog).toBeVisible();
+    const menuHeader = priorityDialog.locator('h4.modal-title');
+    await expect(menuHeader).toHaveText('Message priority');
+
+    // # Verify Standard option is selected by default
+    const standardOption = priorityDialog.getByRole('menuitem', { name: 'Standard' });
+    await expect(standardOption).toBeVisible();
+    await expect(standardOption.locator('svg.StyledCheckIcon-dFKfoY')).toBeVisible();
+
+    // # Close menu and post message
+    await page.getByRole('button', { name: 'Cancel' }).click();
```

Comment:
Test failed here since there's no "Cancel" button. To close, it could be:
```suggestion
    await channelPage.centerView.postCreate.priorityButton.click();
```